infor-design/enterprise-wc-examples
# 1046 - Update settings on existing IdsButton examples

**What's in the PR:**

This PR simply updates the existing examples of IdsButton/related components to support the settings changes in these PRs from the WC repo:

- https://github.com/infor-design/enterprise-wc/pull/1066
- https://github.com/infor-design/enterprise-wc/pull/1080
- https://github.com/infor-design/enterprise-wc/pull/1086

This PR doesn't add any new examples (we are still missing destructive states), but just brings what we currently ship in line with new designs/usage guidelines

**Related issues:**

- infor-design/enterprise-wc#839
- infor-design/enterprise-wc#1046

**How to test:**

- Use `npm run publish:link` to build a local copy of the current [IDS Enterprise Web Components repo](https://github.com/infor-design/enterprise-wc).
- In the example library you want to test (Angular/React/Sveltekit), run `npm link ids-enterprise-wc` to use this local copy instead of the beta from NPM
- Use `npm start` to try each repo.  Use the list of examples below to smoke test each respective framework.  All button examples should reflect the new IDS designs, and should display their icon/text properly (before removal of slot names, all the examples were displaying an "empty" but present button)

Angular: 

- http://localhost:4200/ids-button/example
- http://localhost:4200/ids-app-menu/example
- http://localhost:4200/ids-app-menu/example-footer
- http://localhost:4200/ids-app-menu/sandbox
- http://localhost:4200/ids-breadcrumb/toolbar
- http://localhost:4200/ids-card/footer
- http://localhost:4200/ids-checkbox/example
- http://localhost:4200/ids-empty-message/example
- http://localhost:4200/ids-expandable-area/example
- http://localhost:4200/ids-lookup/example
- http://localhost:4200/ids-masthead/example
- http://localhost:4200/ids-menu-button/example
- http://localhost:4200/ids-menu-button/disabled
- http://localhost:4200/ids-menu-button/display-selected
- http://localhost:4200/ids-modal/example
- http://localhost:4200/ids-modal/focus
- http://localhost:4200/ids-modal/fullsize
- http://localhost:4200/ids-modal/keep-open
- http://localhost:4200/ids-modal/nested
- http://localhost:4200/ids-modal/visible
- http://localhost:4200/ids-popup/example
- http://localhost:4200/ids-progress-bar/example
- http://localhost:4200/ids-radio/example
- http://localhost:4200/ids-search-field/example
- http://localhost:4200/ids-textarea/example
- http://localhost:4200/ids-toast/example
- http://localhost:4200/ids-toast/sandbox
- http://localhost:4200/ids-toggle-button/example
- http://localhost:4200/ids-toggle-button/icon-align-right
- http://localhost:4200/ids-toolbar/example
- http://localhost:4200/ids-toolbar/sandbox
- http://localhost:4200/ids-toolbar/overflowed
- http://localhost:4200/ids-tree/example
- http://localhost:4200/ids-upload-advanced/sandbox

The button examples in the React/Sveltekit example projects have also been updated, however I wasn't able to run them due to other build-related issues.
